apps/mobile/src/components/PostCard/PostHeader/PostHeader.js

```diff
@@ -41,14 +41,14 @@ export default function PostHeader ({
   const [{ currentGroup }] = useCurrentGroup()
   const handleShowMember = () => showMember && showMember(creator.id)
 
-  const creatorHasResponsibility = useHasResponsibility({ person: creator, forCurrentGroup: true })
+  // const creatorHasResponsibility = useHasResponsibility({ person: creator, forCurrentGroup: true })
```

Comment:
Please remove these and all remaining commented-out code if it is not longer used.